### PR TITLE
FIP-54,63, Maintenance upgrades

### PR DESCRIPTION
**Pull request by Joeysantoro**
1. make a new Optimistic 24h Timelock with guardian and OA as admins with a single role to bribe on votium
2. Skimmer for both PSMs (ETH and DAI)
3. Rari infra funding if it passes (TRIBE + FEI 4m over 2 years linear)
4. FIP-54 (Permanently revoke burner, RevokePermissions.sol audited by Dili)

### CODE REVIEW

File: proposals/dao/fip_54.ts [hunk 1]
```diff
@@ -52,10 +104,16 @@ export const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, con
 };
 
 export const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts) => {
-  const { fei, core, restrictedPermissions } = contracts;
+  const { fei, tribe, core, restrictedPermissions, votiumBriberD3pool } = contracts;
```
Comment (ElliotFriedman) [inline proposals/dao/fip_54.ts:107]: validate should check that the skimmers are all pointed to the correct PSM's

also, this pr needs to include the FeiSkimmer.sol file as it is not present

**Review by ElliotFriedman — CHANGES_REQUESTED** [review]

File: test/integration/tests/dependencies.ts [hunk 2]
```diff
@@ -61,6 +62,11 @@ describe('e2e-dependencies', function () {
         const contracts = proposals[proposalName].affectedContractSignoff;
         const deprecated = proposals[proposalName].deprecatedContractSignoff;
 
+        if (proposals[proposalName].deploy) {
+          // Skip these checks if not mainnet deployed
+          doLogging && console.log(`Skipping: ${proposalName}`);
```
Comment (ElliotFriedman) [inline test/integration/tests/dependencies.ts:67]: glad this check got added in

**Comment by ElliotFriedman** [comment]
Everything looks good, DAO, IT and timelock changes all seem minor.

File: proposals/dao/fip_54.ts (hunk 1 above)
Comment (Joeysantoro) [reply proposals/dao/fip_54.ts:107]: manually confirmed after deploy

**Review by ElliotFriedman — APPROVED** [review]
LGTM!

**Comment by ElliotFriedman** [comment]
I would have preferred dao script did more validations on the FEI skimmer targets

**Pull request merged**